### DISCUSSION TITLE
section 03 - Computer vision - Non-linearity (model_1)

### DISCUSSION THREAD

**Discussion by BahryJarbou** [mrdbourke/pytorch-deep-learning#575] [Q&A]
Hi Daniel and everyone :)

A quick heads up on why the model did worse on the gpu compared to the cpu:

I think what caused it is that the layer_stack had the ReLU layer applied to the final linear layer and so the CrossEntropyLoss didn't have logits as inputs. you can test it and see that loss is greater than 1 which violates the range for the cross-entropy loss.

Cheer and thank you for the awesome course ^^
